context_17_test.go

```diff
@@ -7,11 +7,14 @@
 package gin
 
 import (
+	"bytes"
 	"net/http"
 	"net/http/httptest"
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/assert"
+	"golang.org/x/net/context"
```

Comment:
why not use `context`?

Reply:
Because my IDE thought that it was the best fit when autocompleting the import...
Thanks, fixed.